src/QueryM/Ogmios.purs

```diff
@@ -413,7 +419,7 @@ instance Show Epoch where
 
 newtype EraSummaryParameters = EraSummaryParameters
   { epochLength :: EpochLength -- 0-18446744073709552000 An epoch number or length.
-  , slotLength :: SlotLength -- <= 18446744073709552000 A slot length, in seconds.
+  , slotLength :: SlotLength -- <= 9,007,199,254,740,992 A slot length, in miliseconds.
```

Comment:
I think we should delete the comment about the range here since it's just the `Number` maximum rather than a Cardano/Ogmios constraint.

Reply:
We should add a comment mentioning both this constant and MAX_SAFE_INTEFER

Reply:
I will leave the MAX_SAFE_INTEGER then, the previous constant isn't there anymore in Ogmios : 

https://github.com/CardanoSolutions/ogmios/commit/8121673c7f266550e7b4f3f66c6e3938a6b78899